app/components/treatment/Treatment.js

```diff
@@ -1,13 +1,21 @@
 import React, { Component } from 'react'
```

Comment:
Oof, this one is not coming together. Missing a bracket or something somewhere. Will keep at it.